airflow/migrations/versions/0147_2_10_0_rename_dagrun_last_scheduling_decision.py

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Rename dagrun.last_scheduling_decision.
+
+Revision ID: 840479e61d99
+Revises: d482b7261ff9
+Create Date: 2024-06-18 08:58:10.356565
+
+"""
+
+from __future__ import annotations
+
+import sqlalchemy as sa
+from alembic import op
+
+from airflow.utils import timezone
+from airflow.utils.sqlalchemy import UtcDateTime
+
+# revision identifiers, used by Alembic.
+revision = "840479e61d99"
+down_revision = "d482b7261ff9"
+branch_labels = None
+depends_on = None
+airflow_version = "2.10.0"
+
+
+def upgrade():
+    """Apply Rename dagrun.last_scheduling_decision."""
+    with op.batch_alter_table("dag_run", schema=None) as batch_op:
+        batch_op.add_column(sa.Column("next_schedulable", UtcDateTime(), default=timezone.utcnow()))
```

Comment:
This sets the default value to the time whenever the migration is run - is that what we want?

If it is lemme a comment saying why as it looks wrong otherwise

Reply:
It doesn't apply when migration is run. It's the python side stuff. The one that would do that is server_default. This is just a python side thing that will apply when a new object is created. We can remove this anytime in the future without migration

Reply:
You don't want to call it, right?

I'll second that if this is right, we should add a comment. But it's likely splitting hairs because we don't actually do anything with this in the migration. Still though 🤷.

Reply:
If this is not the server side default then it shouldn't be in the migration -- it doesn't do anything and looks confusing having it here.